session/session.go

```diff
@@ -153,17 +153,6 @@ func (s Session) TableName(ctx context.Context) string {
 	return "sessions"
 }
 
-func (s Session) MarshalJSON() ([]byte, error) {
```

Comment:
I think this is a cleaner solution than setting the field upon loading it from the database, because expiry is time based, and we always want the "freshest" value for `Active`. Also a bit confused as to why no tests are failing if this is removed

Reply:
This was removed because it alters the value of list items based on Identity state after the listing query has run. 

As in, a request for `?active=true` yields active session records but then this code block alters the value. This may result in result set with `active: false` due to `s.Active && s.ExpiresAt.After(time.Now()) && (s.Identity == nil || s.Identity.IsActive())`, which will confuse the API consumer. To try and solve this problem, I've made corrections to the query in https://github.com/ory/kratos/pull/2958/commits/eddece5b94834a4d559e9ca8bece589408a782d6